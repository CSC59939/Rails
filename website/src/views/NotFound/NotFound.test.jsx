import React from 'react';
import renderer from 'react-test-renderer';
import { NotFound } from '..';
import '../../utils/tests/test.css';

describe('NotFound View', () => {
  it('Expect the NotFound page to match snap shot', () => { // eslint-disable-line no-undef
    const component = renderer.create(
      <NotFound />,
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot(); // eslint-disable-line no-undef
  });
});
