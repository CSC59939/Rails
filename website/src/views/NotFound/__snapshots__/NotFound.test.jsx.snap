// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound View Expect the NotFound page to match snap shot 1`] = `
<div
  className="home"
>
  <h1
    className="title"
  >
    Rails
  </h1>
  <h5
    className="tag-line"
  >
    You've fallen off track
  </h5>
  <h2
    className="subTitle"
  >
    404 Page Not Found
  </h2>
  <a
    className="ant-btn home-button"
    href="/"
    onClick={[Function]}
  >
    <span>
      Get Back on track
    </span>
    <i
      className="anticon anticon-right-circle"
    >
      <svg
        aria-hidden="true"
        className=""
        data-icon="right-circle"
        fill="currentColor"
        height="1em"
        viewBox="64 64 896 896"
        width="1em"
      >
        <path
          d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm154.7 454.5l-246 178c-5.3 3.8-12.7 0-12.7-6.5v-46.9c0-10.2 4.9-19.9 13.2-25.9L566.6 512 421.2 406.8c-8.3-6-13.2-15.6-13.2-25.9V334c0-6.5 7.4-10.3 12.7-6.5l246 178c4.4 3.2 4.4 9.8 0 13z"
        />
      </svg>
    </i>
  </a>
</div>
`;
